src/StudyViewer/reducers.js

```diff
@@ -10,6 +10,9 @@ const study = (state = {}, action) => {
     return { ...state, docData: action.fileData };
   case 'RECEIVE_OBJECT_FILE_DATA':
     return { ...state, fileData: action.fileData };
+  case 'FILE_DATA_ERROR':
+    console.log(action.error);
```

Comment:
i don't know if we'd want to write it to the console. if the field is optional, omitting it shouldn't trigger an error

Reply:
oops.. that is some leftovers for testing only, will clean it up 🤦